### PR TITLE
MISC: loose ends from root cleanup

### PR DESCRIPTION
**Pull request by ebolyen**
Forgot to update the relative links in some of the dev readmes, and the `manifest.in` file no longer includes the dropped elements (or things that just don't need to be shipped in an sdist).

Also, it turns out renaming `COPYING.txt` to `LICENSE.txt` means the headers were wrong, fortunately it was quick to use sed to make the update.

### CODE REVIEW

**Comment by ebolyen** [comment]
Coveragerc has been moved into pyproject.toml

**Comment by ebolyen** [comment]
Coveragerc has been restored

File: pyproject.toml [hunk 1]
```diff
@@ -5,3 +5,14 @@ requires = ["Cython>=0.29.32", "numpy>=1.9.2", "setuptools", "wheel"]
...
+
+[tool.check-manifest]
+ignore = [
+    ".coveragerc",
+    ".dockerignore",
+    ".editorconfig",
+    "RELEASE.md",
+    "REVIEWING.md",
+    "aarch64.Dockerfile",
+    "checklist.py",
```
Comment (gregcaporaso) [inline pyproject.toml:17]: I think `PULL_REQUEST_TEMPLATE.md` should also be in this list.

**Review by gregcaporaso — APPROVED** [review]
Just one comment - everything else looks good to me!

File: pyproject.toml (hunk 1 above)
Comment (ebolyen) [reply pyproject.toml:17]: That's already ignored by being in .github (which is a default exclusion for check-manifest)

Comment (gregcaporaso) [reply pyproject.toml:17]: Got it, all good then.

**Pull request merged**